src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -651,21 +651,29 @@ function disableRow(elem, row) {
     $tdElements.parent().hide();
 }
 
-function feedbackRecipientUpdateVisibilityOptions(elem) {
-    elem = $(elem);
-    if (elem.val() === 'OWN_TEAM' || elem.val() === 'TEAMS' ||
-            elem.val() === 'INSTRUCTORS' || elem.val() === 'OWN_TEAM_MEMBERS') {
-        enableRow(elem, 1);
-        disableRow(elem, 3);
+function feedbackRecipientUpdateVisibilityOptions(recipientType) {
+    $recipientType = $(recipientType);
```

Comment:
Hmm, perhaps the reasoning that "when this method is called, a variable named recipientType will be passed as parameter" is not a good way of reasoning about what the parameter should be called. 

Honestly, if I were to maintain this code, I rather have the the old <code>elem</code> over recipientType, recipientType may make sense in the calling function given enough context, but here, this parameter is a html element that's passed in. Let's just not touch this part of the code, I guess, since it's not actually related to this issue anyway?